cs3api4lab/auth/__init__.py

```diff
@@ -0,0 +1,3 @@
+from cs3api4lab.auth.eos import Eos
```

Comment:
Don't you need to import `check_auth_interceptor` here?

In the files and shares api you do `from cs3api4lab.auth import check_auth_interceptor`, which looks like it will fail.

Reply:
I add this import and fix broken dependency to "pyjwt".
But, the plugin works without this import when I tested and local instance.